src/entities/WeeklyDebrief.ts

```diff
@@ -0,0 +1,42 @@
+import { Field, ID, ObjectType } from 'type-graphql';
+import { prop as Property, DocumentType } from '@typegoose/typegoose';
+import mongoose from 'mongoose';
+import { Article } from './Article';
+
+@ObjectType({ description: 'The Weekly Debrief Model' })
+export default class WeeklyDebrief {
+  @Field(() => ID)
+  id: string;
+
+  @Field()
+  @Property()
+  uuid: string;
+
+  @Field()
+  @Property()
+  creationDate: Date;
+
+  @Field()
+  @Property()
+  expirationDate: Date;
+
+  @Field()
+  @Property({ default: 0 })
+  numShoutouts?: number;
+
+  @Field()
+  @Property({ default: 0 })
+  numBookmarkedArticles?: number;
+
+  @Field()
+  @Property({ default: 0 })
+  numReadArticles?: number;
+
+  @Field((type) => [Article])
+  @Property({ required: true, type: () => Article, default: [] })
+  readArticles: mongoose.Types.DocumentArray<DocumentType<Article>>;
+
+  @Field((type) => [Article])
+  @Property({ required: true, type: () => Article, default: [] })
+  randomArticles: mongoose.Types.DocumentArray<DocumentType<Article>>;
```

Comment:
Maybe some documentation on what randomArticles is? I'm assuming that we randomize once and then store those two random picks but I also don't see why not do something like using a seed that we can guarantee getting the same random articles back (for example, the seed used to get the random articles for a user will always be their ID or, to make it more random, will be the ID of the weekly debrief).

Reply:
The MVP avoids dealing with this kind of detail. We still don't even know if users care about random articles yet.